src/components/spinLoader/index.ts

```diff
@@ -0,0 +1,5 @@
+import { SpinLoader } from './spinLoader';
+
+export { SpinLoader };
+
+export default SpinLoader;
```

Comment:
Do we plan to export this component as a part of the library?
If yes, it should be defined in Design System with consistent name and covered with storybook stories for visibility for UX team.